local.go

```diff
@@ -35,7 +38,7 @@ func digest(URL string) (digest string, err error) {
 	if err != nil {
 		return
 	}
-	defer f.Close()
+	defer func() { _ = f.Close() }()
```

Comment:
The errcheck linter is a really good one to satisfy though :)

But yeah, adding util methods that don't return error and using those is a better way to satisfy the errcheck and say, I don't care if it errors.